static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,121 @@
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Tracked data is stored in a **centralized** remote location, with the ability
+  to create distributed copies on other remotes.
+- Several projects can **share** the same files, guaranteeing that everyone has
+  access to the same data versions. See
+  [Share Data and Model Files](/doc/use-cases/share-data-and-model-files) for
+  more information.
+- Projects that import data from the registry don't need to push these large
+  files to their own [remotes](/doc/command-reference/remote), **saving space**
+  on storage – they may not even need a remote at all, using only their local
+  <abbr>cache</abbr>.
+- DVC data registries can handle multiple versions of data and ML modes with a
+  familiar CLI. See
+  [Data and Model Files Versioning](/doc/use-cases/data-and-model-files-versioning)
+  for more information.
+- DVC data registries are versioned with Git, so you can always track the
+  history of the project the same as you manage your source code repository.
+
+A possible risk of shared data registries is that, if the source project or its
+remote storage are lost for any reason, several other projects depending on it
+may stop being reproducible. So this strategy is best when the registry is owned
+and controlled internally by the same team as the projects that employ it.
+
+## Using properly versioned registries
+
+In the [Versioning Tutorial](/doc/tutorials/versioning) we use a ZIP file
+containing images of cats and dogs, and then we update the dataset with more
+images from a second ZIP file. These compressed archives are downloaded from our
+own
+[iterative/dataset-registry](https://github.com/iterative/dataset-registry)), a
+<abbr>DVC project</abbr> hosted on GitHub using `dvc get`. They can be found as
+<abbr>outputs</abbr> of 2 separate
+[DVC-files](/doc/user-guide/dvc-files-and-directories) in the `tutorial/ver`
+directory.
+
+There are a few problems with the way this dataset is structured (in 2 parts, as
```

Comment:
I think we still put too many details on this "improper" structure

Reply:
OK, I'll summarize more, along the lines of your other review comments.